src/components/CardTable/CardTable.jsx

```diff
@@ -109,14 +109,16 @@ export default class CardTable extends Component {
   }
 
   get largestTableWidth () {
-    const tableNodes = Object.values(this.tableNodes)
+    const tableNodes = Object.values(this.tableNodes || {})
     const tableWidths = tableNodes.map(tableNode => tableNode.clientWidth)
 
     return Math.max(...tableWidths)
   }
 
   get parentWidth () {
-    return getElementContentWidth(this.containerNode.parentNode)
+    const ZERO = 0
+
+    return this.containerNode ? getElementContentWidth(this.containerNode.parentNode) : ZERO
```

Comment:
The reason is overwrites with null is because the `containerNode` ref function is not defined in the class which produces a null followed by the node every render. I would suggest you turn that ref function into a class function and then also instantiate `containerNode` in the class.